lib/ExpensiMark.d.ts

```diff
@@ -57,13 +66,14 @@ export default class ExpensiMark {
      *
      * @param htmlString
      */
-    htmlToMarkdown(htmlString: string): string;
+    htmlToMarkdown(htmlString: string, personalDetails: PersonalDetails): string;
```

Comment:
It could be treated as metadata, allowing us to pass various data through the second parameter.

E.g. 
const metadata = { personalDetails }
htmlToMarkdown(html, metadata)

const metadata = { reportDetails }
htmlToMarkdown(html, metadata)

same in htmlToText.
WDYT?

Reply:
Should the second parameter be optional? 

```suggestion
    htmlToMarkdown(htmlString: string, personalDetails?: PersonalDetails): string;
```